app/src/main/java/com/github/sdpcoachme/profile/ProfileActivity.kt

```diff
@@ -194,6 +221,31 @@ class ProfileActivity : ComponentActivity() {
         ) {
             TitleRow(userInfo, isViewingCoach)
             Spacer(modifier = Modifier.height(10.dp))
+            if (currIsCoach || isViewingCoach) {
```

Comment:
with this implementation, it shows a rating even for non-coaches that are displayed via the ProfileActivity with isViewingCoach set to true (the default behavior through out the app). Would probably make more sense to only display the rating when currIsCoach (or userInfo.coach) without taking into account whether isViewingCoach is true or not.